services/search/README.md

```diff
@@ -1,34 +1,56 @@
 # Search
 
-The search service is responsible for metadata and content extraction, stores that data as index and makes it searchable. The following clarifies the extraction terms _metadata_ and _content_:
+The search service is responsible for metadata and content extraction,
+the retrieved date is indexed and made searchable.
 
-*   Metadata: all data that _describes_ the file like `Name`, `Size`, `MimeType`, `Tags` and `Mtime`.
-*   Content: all data that _relates to content_ of the file like `words`, `geo data`, `exif data` etc.
+The search service runs out of the box with the shipped default `basic` configuration.
+No further configuration is needed.
 
-## General Considerations
+## Search backends
 
-*   To use the search service, an event system needs to be configured for all services like NATS, which is shipped and preconfigured.
-*   The search service consumes events and does not block other tasks.
-*   When looking for content extraction, [Apache Tika - a content analysis toolkit](https://tika.apache.org) can be used but needs to be installed separately.
+To store and query the indexed data, a search backend is needed.
 
-Extractions are stored as index via the search service. Consider that indexing requires adequate storage capacity - and the space requirement will grow. To avoid filling up the filesystem with the index and rendering OpenCloud unusable, the index should reside on its own filesystem.
+As of now, the search service supports the following backends:
 
-You can change the path to where search maintains its data in case the filesystem gets close to full and you need to relocate the data. Stop the service, move the data, reconfigure the path in the environment variable and restart the service.
+- [bleve](https://github.com/blevesearch/bleve) (default)
+- [opensearch](https://opensearch.org/)
 
-When using content extraction, more resources and time are needed, because the content of the file needs to be analyzed. This is especially true for big and multiple concurrent files.
+### Bleve
 
-The search service runs out of the box with the shipped default `basic` configuration. No further configuration is needed, except when using content extraction.
+Bleve is a lightweight, embedded full-text search engine written in Go and is the default search backend.
+It is easy to set up and requires no additional services to run.
```

Comment:
Since we're listing the supported env vars for the opensearch backend, should we list the bleve one here as well (only `SEARCH_ENGINE_BLEVE_DATA_PATH` afaik)?